packages/xdl/src/Webpack.ts

```diff
@@ -296,7 +341,8 @@ function transformCLIOptions(options: CLIWebOptions): BundlingOptions {
 
 async function createWebpackConfigAsync(
   projectRoot: string,
-  options: CLIWebOptions = {}
+  options: CLIWebOptions = {},
+  unimodules: boolean = false
```

Comment:
It's usually best to avoid positional boolean arguments because they can lead in code that's harder to read because it's not possible to guess what each argument means from a function call like `createWebpackConfigAsync(projectRoot, options, true, false)`.

You can add a `unimodules` property to `options` instead.